subject_behavdata.py

```diff
@@ -2,273 +2,414 @@
+    cols = ['trial_number', 'onset', 'duration', 'offset', 'trial_type',
+            'response', 'response_time', 'stim_id', 'stim_file',
+            'stim_category', 'stim_name', 'recognition_accuracy',
+            'recognition_responsetime', 'position_correct',
+            'position_response', 'position_accuracy', 'position_responsetime']
     mainEnc = mainEnc[cols]
-    #Re-merge mainEnc and mainCTL and re-order by trial number
+    # Re-merge mainEnc and mainCTL and re-order by trial number
     mainMerged = mainEnc.append(mainCTL, ignore_index=True)
-    mainMerged.sort_values('trial_number', axis=0, ascending=True, inplace=True)
+    mainMerged.sort_values('trial_number', axis=0, ascending=True,
+                           inplace=True)
     return mainMerged
 
+
 def extract_taskFile(bID, sID, file_list, output):
-    #import data from three text files into pandas DataFrames
+    """
+    Parameters:
+    ----------
+    bID: string (behavioral ID)
+    sID: string (structural ID)
+    file_list: list (three input files)
+    output: string (output Folder)
+
+    Return:
+    ----------
+    None
+    """
+    # import data from three text files into pandas DataFrames
     encMain = pd.read_csv(file_list[0], sep='\t')
     manualEdits = ['3303819', '5477234', '6417837', '7674650']
     if bID in manualEdits:
         encOnsets = pd.read_csv(file_list[1], sep='\t', header=None)
-    else:     
+    else:
         encOnsets = pd.read_fwf(file_list[1], infer_nrows=210,
-        delim_whitespace=True, header=None)
-    retriev = pd.read_csv(file_list[2], sep='\t', encoding = 'ISO-8859-1')
-    #clean up each file
+                                delim_whitespace=True,
+                                header=None)
+    retriev = pd.read_csv(file_list[2], sep='\t', encoding='ISO-8859-1')
+    # clean up each file
     encMain = cleanMain(encMain)
     encOnsets = cleanOnsets(encOnsets)
     retriev = cleanRetriev(retriev)
-    #import onset times from encOnset into encMain
+    # import onset times from encOnset into encMain
     encMain = addOnsets(encMain, encOnsets)
-    #import post-scan performance data from retriev into encMain
+    # import post-scan performance data from retriev into encMain
     encMain = addPostScan(encMain, retriev)
-    #export encMain and retriev into tsv files (output directorty)
-    #encMain.to_csv(output+'/TaskFile_bID'+bID+'_mriID'+sID+'.tsv',
-    #sep='\t', header=True, index=False)
+    # export encMain and retriev into tsv files (output directorty)
+    # encMain.to_csv(output+'/TaskFile_bID'+bID+'_mriID'+sID+'.tsv',
+    # sep='\t', header=True, index=False)
     encMain.to_csv(output+'/sub-'+sID+'_ses-4_task-memory_events.tsv',
-    sep='\t', header=True, index=False)
+                   sep='\t', header=True, index=False)
     retriev.to_csv(output+'/PostScanBehav_bID'+bID+'_mriID'+sID+'.tsv',
-    sep='\t', header=True, index=False)
```

Comment:
Change line 375 to:
retriev.to_csv(output+'/PostScanBehav_pscid'+bID+'_dccid'+sID+'.tsv',